lib/gollum-lib/filter/tags.rb

```diff
@@ -98,10 +109,9 @@ def process_tag(tag)
   #   if it is not.
   #
   def process_include_tag(tag)
-    return unless /^include:/.match(tag)
+    return html_error('Cannot process include directive: no page name given') if tag.length <= 8
     page_name          = tag[8..-1]
-    resolved_page_name = ::File.expand_path(page_name, "/"+@markup.dir)
-
+    resolved_page_name = ::File.expand_path(page_name, "#{::File::SEPARATOR}#{@markup.dir}")
```

Comment:
Not sure. I tried putting a `::File.join` here but that broke some tests. The point here is to prevent people from including `/etc/password`. `File.expand_path` here never returns a path lower in the directory tree than `@markup.dir`. Does `Pathname` have a similar method?